### DISCUSSION TITLE
CPU Usage VERY high in new version

### DISCUSSION THREAD

**Discussion by gweddlenz** [Shubhamai/StableView#78] [General]
Hi there,

Me again. I didn't pick this up before, but I've just found the latest version (and beta you sent me) is killing my CPU. It using 70% CPU by itself. Old version without face recapture uses about 2% CPU. Any ideas?

Thanks,
Gary

**Comment by gweddlenz** [comment]
Pulling FPS slider to lowest setting uses about 55% CPU. Still very high.

CPU = i5-12400F. Windows 10. 32G RAM.

**Comment by Shubhamai** [comment]
Ah, I failed to notice this too, Working on the fix.

**Comment by Shubhamai** [comment]
Hi @gweddlenz , just fixed it in the recent [pre-release version](https://github.com/Shubhamai/StableView/releases/tag/v1.1.0), it is showing about 6-10% on my machine.

**Comment by gweddlenz** [comment]
Perfect, I'll give it a shot!

Cheers,
Gary
